config/config.go

```diff
@@ -42,11 +43,13 @@ func Initialize() error {
 		Debug:       strings.ToLower(os.Getenv(EnvDebug)) == "true",
 	}
 	if clusterName := os.Getenv(EnvClusterName); len(clusterName) > 0 {
-		cfg.ClusterName = clusterName
+		cfg.AutodiscoveryTags = fmt.Sprintf("k8s.io/cluster-autoscaler/%s=owned,k8s.io/cluster-autoscaler/enabled=true", clusterName)
+	} else if autodiscoveryTags := os.Getenv(EnvAutodiscoveryTags); len(autodiscoveryTags) > 0 {
+		cfg.AutodiscoveryTags = autodiscoveryTags
 	} else if autoScalingGroupNames := os.Getenv(EnvAutoScalingGroupNames); len(autoScalingGroupNames) > 0 {
 		cfg.AutoScalingGroupNames = strings.Split(strings.TrimSpace(autoScalingGroupNames), ",")
 	} else {
-		return fmt.Errorf("environment variables '%s' or '%s' are not set", EnvAutoScalingGroupNames, EnvClusterName)
+		return fmt.Errorf("environment variables '%s', '%s' or '%s' are not set", EnvAutoScalingGroupNames, EnvClusterName, EnvAutodiscoveryTags)
 	}
```

Comment:
Could you do something to test this logic? 
Sorry for asking so much of you -- I haven't modified this app in a while, and I just want to be 100% sure that we're not introducing any regression, both now and in the future 🙏 

If you'd rather not, then it's alright, I can take care of it when I have some time :)

Reply:
Its ok, there you go https://github.com/TwiN/aws-eks-asg-rolling-update-handler/pull/21/commits/57d60619fb32e0c2db0a48d12cd2b09d1f59171d 🤗

Reply:
Awesome! Thank you so much 🥳